app/javascript/application.js

```diff
@@ -0,0 +1,19 @@
+import "@hotwired/turbo-rails";
+import "./controllers";
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from "react-router-dom";
+import { Provider } from 'react-redux';
+import store from "./redux/store";
+import App from "./components/App";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
```

Comment:
![image](https://github.com/dazehinn/hello-rails-react/assets/93039359/135e2389-5e7e-4b5f-a047-6bf7dfa8e46b)

- [ ] Great work so far :+1: but it would be nice if you fix the error which is coming from your react app :fearful: Kindly fix it to make your project work as expected :blush: